src/pages/iou/IOUCurrencySelection.js

```diff
@@ -117,9 +112,7 @@ function IOUCurrencySelection(props) {
                 ? []
                 : [
                       {
-                          title: translate('iOUCurrencySelection.allCurrencies'),
```

Comment:
It looks like this text was removed, but the strings still exist in the `en.ts` and `es.ts` files. Should it be removed from there too?

Reply:
If so, I can remove it as part of my PR here https://github.com/Expensify/App/pull/29876/ but I just want to be sure it's OK to do so.

Reply:
Ah yeah, we didn't notice, if they are not used anymore, let's remove them